client/app/components/Footer/index.js

```diff
@@ -4,115 +4,115 @@ import Reveal from 'react-reveal';
+          <div className={`column content has-text-left is-3 is-offset-1 ${styles.darkTextFix}`}>
+            <h3>Quick Links</h3>
+            <p>
+              <Link to={'/'}>Home</Link>
+            </p>
+            <p>
+              <Link to={'/solutions'}>Solutions</Link>
+            </p>
+            <p>
+              <Link to={'/resources'}>Resources</Link>
+            </p>
+            <p>
+              <Link to={'/contact-us'}>Contact Us</Link>
+            </p>
+            {/* Bring social media icons closer together and add "follow us on:"*/}
+            <hr className={`has-text-left ${styles.hrLeft}`} />
+            <div style={{ display: 'flex', justifyContent: 'center' }}>
+              <div style={{ flex: 1, padding: '5px' }}>
+                <a href="https://www.facebook.com/IDICIA/">
+                  <i
+                    className="fa fa-facebook-official fa-2x"
+                    aria-hidden="true"
+                  />
+                </a>
+              </div>
+              <div style={{ flex: 1, padding: '5px' }}>
+                <a href="https://twitter.com/4idicia?lang=en">
+                  <i className="fa fa-twitter-square fa-2x" aria-hidden="true" />
+                </a>
+              </div>
+              <div style={{ flex: 1, padding: '5px' }}>
+                <a href="https://www.linkedin.com/company/idicia">
+                  <i className="fa fa-linkedin-square fa-2x" aria-hidden="true" />
+                </a>
+              </div>
             </div>
+          </div>
 
-              {/* logo and mission statement */}
-            <div className={`column has-text-centered is-4 ${styles.darkTextFix}`}>
-                <img className={styles.footerLogo} src={LargeLogo} alt="Idicia Footer Logo" />
-                <hr/>
-                <p>Integrated Verification, Enrichment and Comparison Solutions for Identity Verification, Fraud Prevention and Compliance.</p>
-                <table className={`table is-narrow ${styles.contactTable}`}>
-                  {/* How to get these two to sit on the same line?! */}
-                  <td className={`${styles.info}`} >
-                    <a href="mailto:info@idicia.com">info@idicia.com</a>
-                  </td>
-                  <td className={`${styles.info}`} >1.844.4IDICIA</td>
-                </table>
-            </div>
+          {/* logo and mission statement */}
+          <div className={`column has-text-centered is-4 ${styles.darkTextFix}`}>
+            <img
+              className={styles.footerLogo}
+              src={'http://res.cloudinary.com/dyjo5a3ci/image/upload/v1514087342/idicia/white.png'}
+              alt="Idicia Footer Logo"
+            />
+            <hr />
+            <p>Integrated Verification, Enrichment and Comparison Solutions for Identity Verification, Fraud Prevention and Compliance.</p>
+            <p className={styles.contactEmailAndPhone}>
```

Comment:
@leseanbiz Refer to this and the relevant css, does that answer your question about how to get the two next to each other without a table?

@Globolobo